package.json

```diff
@@ -255,7 +255,7 @@
     "redux-thunk": "2.3.0",
     "reselect": "4.0.0",
     "rst2html": "github:thoward/rst2html#990cb89",
-    "rxjs": "6.4.0",
+    "rxjs": "7.0.0-alpha.0",
```

Comment:
Any reason we are upgrading to 7.0.0-alpha instead of continue on the stable 6.x version? I'm just thinking that it might be a bit risky to run an alpha version if we really don't need it.

Reply:
Because cancellation didn't work properly as in the Observer was unsubscribed but the cancellation of the fetch request was not due to some webpack issue resolved [here](https://github.com/ReactiveX/rxjs/pull/5001)

Reply:
7.0.0-alpha.1 is also out

Reply:
Yes, this change looks interesting https://github.com/reactivex/rxjs/commit/0e4849a don't know about the rest. But I think we're gonna hold this PR until we branch of 6.6 so maybe the 7.0 release will be out by then?